src/types/index.ts

```diff
@@ -529,6 +529,14 @@ export type ITransactionsState = {
 
 export type IAccountTransactionsState = Record<string, ITransactionsState>
 
+export type FeeItem = {
```

Comment:
I know there is not much of a difference between type and interface but in most cases our app uses interfaces prefixed with "I". Maybe this one could follow this to be more consistent?